utils/unit_test/CMakeLists.txt

```diff
@@ -0,0 +1,104 @@
+############################################################################
```

Comment:
We don't need to do it in this PR but we should use this new utils package as an opportunity to unify our test harness and have a single version sitting in `utils/`

Reply:
Done